src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -806,7 +800,15 @@ public function dumpString(Cursor $cursor, string $str, bool $bin, int $cut)
     {
         if ('' === $str && isset($cursor->attr['img-data'], $cursor->attr['content-type'])) {
             $this->dumpKey($cursor);
-            $this->line .= $this->style('default', $cursor->attr['img-size'] ?? '', []).' <samp>';
+            $this->line .= $this->style('default', $cursor->attr['img-size'] ?? '', []);
+            $this->line .= ' <samp';
+            if ($cursor->depth >= $this->displayOptions['maxDepth']) {
+                $this->line .= ' class=sf-dump-compact>';
+            } elseif ($cursor->depth < $this->displayOptions['maxDepth']) {
```

Comment:
is this case needed at all? it looks like an "else" to me

Reply:
You are right. I fixed it.